data/reference/syntax.adoc

```diff
@@ -0,0 +1,276 @@
+= AsciiDoc Syntax
+:icons: font
+:stem:
+:toc: left
+:url-docs: https://asciidoctor.org/docs
+:url-gem: https://rubygems.org/gems/asciidoctor
+
+A brief reference of the most commonly used AsciiDoc syntax.
+See {url-docs} to find the full documentation and reference.
+
+== Paragraphs
+
+A normal paragraph.
+Line breaks do not get preserved.
+// line comments, starting with //, are skipped
+
+A blank line separates paragraphs.
+
+[%hardbreaks]
+This paragraph bears the `hardbreaks` option.
+Notice line breaks are now preserved.
+
+ An indented (literal) paragraph disables text formatting,
+ preserves spaces and line breaks, and is displayed in a
+ monospaced font.
+
+[sidebar#id.role]
+Add a style, ID, and/or role to give a paragraph special meaning, like this sidebar.
+
+NOTE: An admonition paragraph, like this note, grabs the reader's attention.
+
+TIP: You can convert this document using `asciidoctor` to see the published output.
+
+== Text Formatting
+:hardbreaks:
+
+.Constrained (applied at word boundaries)
+*strong importance* (aka bold)
+_stress emphasis_ (aka italic)
+`literal text` (aka monospaced)
++passthrough text+ (substitutions disabled)
+`+literal passthrough text+` (literal text with substitutions disabled)
+
+.Unconstrained (applied anywhere)
+**C**reate+**R**ead+**U**pdate+**D**elete
+fan__freakin__tastic
+``mono``culture
+
+.Replacements
+A long time ago in a galaxy far, far away...
+(C) 1976 Arty Artisan
+I believe I shall--no, actually I won't.
+
+.Macros
+pass:c,a[custom substitutions] (c=specialchars, q=quotes, a=attributes, r=replacements, m=macros, p=post_replacements)
```

Comment:
I think we should use a comment here and below.

Reply:
The reason I didn't was to emphasize this is an inline macro. It also gives context in the published output (which otherwise would just say "custom substitutions"). It's tricky.

Reply:
I see... what about something like:

`The australian flag has pass:c,a[*****] stars on it.`

Reply:
Neat idea! Let me play with that.

Reply:
I made the example more fun and relevant. Also moved the reference to a line comment.

Reply:
For as many times as pass is needed, it's always hard to come up with an example when you want to show it ;)